modules/es-extensions/store/extensions/app/greg-diff/themes/store/css/diff.css

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @thushara35 , I have updated the copyright year in [this commit](https://github.com/wso2/product-greg/pull/854/commits/5f9aea00f943aca4c6b5ce3ca67c1e976cd71ef3)